editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -1252,6 +1252,9 @@ bool TileMapEditor::forward_gui_input(const Ref<InputEvent> &p_event) {
 
 			if (node->get_tileset()->has_tile(tile_under))
 				tile_name = node->get_tileset()->tile_get_name(tile_under);
+			float max_lenght = 12;
```

Comment:
Typo, should be `max_length`. And why use a float?

Reply:
@akien-mga Thank you for pointing out that I was careless

Reply:
Can you amend the commit to fix these issues?